memfs/memory.go

```diff
@@ -20,19 +21,18 @@ import (
 
 const separator = filepath.Separator
 
-var errNotLink = errors.New("not a link")
-
 // Memory a very convenient filesystem based on memory files.
 type Memory struct {
 	s *storage
-
-	tempCount int
 }
 
 // New returns a new Memory filesystem.
 func New() billy.Filesystem {
 	fs := &Memory{s: newStorage()}
-	fs.s.New("/", 0755|os.ModeDir, 0)
+	_, err := fs.s.New("/", 0755|os.ModeDir, 0)
+	if err != nil {
+		log.Printf("failed to create root dir: %v", err)
```

Comment:
Is this intended? You probably want to return the error here

Reply:
LGTM otherwise

Reply:
This was intentional as I didn't want to change the API on this PR, so the error is at least being logged. I was trying to keep the changes to a minimum while fixing the linter errors. The overall API will be revisited in future changes, and this revisited.